scss/elements/_forms.scss

```diff
@@ -131,20 +131,20 @@ input[type="file"] {
 //
 // Styleguide elements.forms.checkboxes
 
-input[type="checkbox"],
-input[type="radio"] {
+[type="checkbox"],
```

Comment:
You were right! I ran into a super annoying thing with specificity and this solved it.